src/twisted/test/proto_helpers.py

```diff
@@ -159,6 +162,7 @@ class StringTransport:
     """
 
     disconnecting = False
+    aborted = False
```

Comment:
This expands the interface of `StringTransport` further in a bad direction - to export an apparently public attribute that isn't part of its interface.

There's already another public attribute you could use for this purpose - `disconnected` - https://github.com/twisted/twisted/blob/aeeffc7a94880f156d30526b73c89348b6948db0/src/twisted/internet/abstract.py#L201  This attribute isn't really documented or explained well either, but at least it sticks to existing crummy interfaces and doesn't drag in a new one just for testing [that we really shouldn't have](https://glyph.twistedmatrix.com/2015/05/separate-your-fakes-and-your-inspectors.html)